test/e2e/e2ethanos/services.go

```diff
@@ -1026,3 +1026,66 @@ func NewS3Config(bucket, endpoint, basePath string) s3.Config {
 		},
 	}
 }
+
+// NOTE: by using aggregation all results are now unsorted.
+var QueryUpWithoutInstance = func() string { return "sum(up) without (instance)" }
+
+// e2ethanos.DefaultPromConfig returns Prometheus config that sets Prometheus to:
+// * expose 2 external labels, source and replica.
+// * optionallly scrape self. This will produce up == 0 metric which we can assert on.
+// * optionally remote write endpoint to write into.
+func DefaultPromConfig(name string, replica int, remoteWriteEndpoint, ruleFile string, scrapeSelf bool, scrapeTargets ...string) string {
```

Comment:
golang document shoud be follow this format:
// DefaultPromConfig is xxx.
https://go.dev/doc/effective_go#commentary

Reply:
Good catch, I missed this during find-and-replace, let me fix it